### ISSUE TITLE
Question: Error returned when using filter with PostgreSQL

### ISSUE THREAD

**Issue by maxosen** [TaKO8Ki/gobang#100]
When using Gobang with PostgreSQL, I encounter the error
```
error returned from database: LIMIT #, # syntax is not supported
```
The screenshot below described my error.

My table `sec_modules` contain the column `id`. And I type the filter expression `id = 23`, when press "Enter", I got the error.

Screenshot of the error:
![image](https://user-images.githubusercontent.com/24519631/133263473-91693f2f-98e4-438c-befe-d627583638ea.png)

I'm using Gobang installed from "gobang-0.1.0-alpha.3-x86_64-pc-windows-msvc.zip", Windows 10, and PostgreSQL 13.2.

Is there any syntax error with my filter expression? Thanks.

**Comment by TaKO8Ki** [comment]
Thank you for opening the issue! This is a bug. I'm going to fix it as soon as possible.

**Comment by TaKO8Ki** [comment]
I've just fixed the bug in https://github.com/TaKO8Ki/gobang/pull/101. I'm going to release v0.1.0-alpha.4 later.

**Comment by TaKO8Ki** [comment]
I've just released [v0.1.0-alpha.4](https://github.com/TaKO8Ki/gobang/releases/tag/v0.1.0-alpha.4). I would be grateful if you could open issues!
